### ISSUE TITLE
Many-Many relation

### ISSUE THREAD

**Issue by lucianopf** [Canner/gqlify#28]
**Describe the bug**
Hello guys, I'm currently developing a new data source for DynamoDB and I encountered a barrier which seems to also happens to other data sources. (I've tested with memory and firebase realtime)

When dealing with many-to-many relations the hook that calls the method `addIdToManyRelation` it is not providing the correct atributes, at lest of of the parameters are always undefined, such as `sourceSideId` or `targetSideId` and by that I'm not able to create a many-many scenario.

Is that already mapped at another issue? If so, I'm sorry for reporting that again. 🤔  

**To Reproduce**
```
// model.graphql
type Book @GQLifyModel(dataSource: "firebase", key: "Book") {
  id: ID! @unique @autoGen
  name: String!
  categories: [Category]
}


type Category @GQLifyModel(dataSource: "firebase", key: "Category") {
  id: ID! @unique @autoGen
  name: String!
  books: [Book]
}
```

Than using either a createBook or updateBook with the create action at the categories that happens.

**Expected behavior**
1 - It should save in a intermediary table (or 2), with the names in lower case and a `_` in between the table names.
2 - It should call this method with all the 4 parameters correctly

**Additional context**
I'm really in love with this project and I'm planing to create a couple more data sources, if you guys need help at the project please shoot me and I'll do my best to contribute to this project!! 

(The only missing key from the DynamoDB data source is the many-many relation =S)

**Comment by wwwy3y3** [comment]
hi @lucianopf thanks for the bug report!

We're working on it right now! Should be fixed in no time

**Comment by lucianopf** [comment]
AWESOME! Thanks for the heads up @wwwy3y3 !!

It just made me believe once again in the project! 😬

**Comment by wwwy3y3** [comment]
Hi @lucianopf 

It should be fixed at `v1.0.10`

Here's the sandbox with your datamodel: https://codesandbox.io/s/5z239jlw6l

If there's anything go wrong again, pls don't hestitate to open an issue.

thanks!
